types/ChatMessage.ts

```diff
@@ -0,0 +1,13 @@
+import Chat from './Chat';
+import User from './User';
+
+export default interface ChatMessage {
+  id: Number;
```

Comment:
Should never use `Number` in typescript. Always use `number` (yes, ts moment)
```suggestion
  id: number;
```